view/frontend/templates/breadcheckout/cartview.phtml

```diff
@@ -189,13 +189,15 @@
                 breadConfig.actAsLabel = false;
             }
 
-            if(cartSizeFinancing.enabled){
-                var cartSizeFinancingId = cartSizeFinancing.id;
-                var cartSizeThreshold = cartSizeFinancing.threshold;
-                var items = document.defaultItems;
+            <?php if($block->isTargetedFinancing() && $block->checkFinancingMode('cart')):?>
+                var cartSizeFinancingId = isFinancing.id;
+                var cartSizeThreshold = isFinancing.threshold;
+                var items = document.minicartItems;
```

Comment:
why is this `minicartItems` here on the main cartview.phtml

Reply:
because I made a mistake :|